config/environment.js

```diff
@@ -42,8 +42,15 @@ module.exports = function(environment) {
     }
 
     if (environment === 'production') {
+        ENV.locationType = 'hash';
+        ENV.rootURL = '/osfpages/';
```

Comment:
May need there in `if (process.env.BACKEND === 'stage'` What happens right now if you try to load the app directly to a different route?